lib/Page.js

```diff
@@ -11,6 +11,10 @@ const FsUtil = require('./util/fsUtil');
 const logger = require('./util/logger');
 const MarkBind = require('./markbind/lib/parser');
 
+const FOOTERS_FOLDER_PATH = '_markbind/footers/';
+
+const FLEX_DIV_HTML = '<div id="flex-div"></div>\n';
```

Comment:
Trailing  `\n` doesn't seem necessary.

Reply:
Assumed that it would make the html output more readable. But everything is handled nicely with `htmlBeautify` 👍. Tested without `\n` and looks fine.